pkg/events/authorization.go

```diff
@@ -1,88 +0,0 @@
-package events
-
-import (
-	"context"
-
-	cs "github.com/cloudtrust/common-service/v2"
-	"github.com/cloudtrust/common-service/v2/log"
-	"github.com/cloudtrust/common-service/v2/security"
-	api "github.com/cloudtrust/keycloak-bridge/api/events"
-)
-
-// Tracking middleware at component level.
-type authorizationComponentMW struct {
-	authManager security.AuthorizationManager
-	logger      log.Logger
-	next        Component
-}
-
-// MakeAuthorizationManagementComponentMW checks authorization and return an error if the action is not allowed.
-func MakeAuthorizationManagementComponentMW(logger log.Logger, authorizationManager security.AuthorizationManager) func(Component) Component {
-	return func(next Component) Component {
-		return &authorizationComponentMW{
-			authManager: authorizationManager,
-			logger:      logger,
-			next:        next,
-		}
-	}
-}
-
-func (c *authorizationComponentMW) GetActions(ctx context.Context) ([]api.ActionRepresentation, error) {
-	var action = security.EVGetActions.String()
```

Comment:
Actions might be removed from common-service? or this code was already moved in another service?

Reply:
Used in event service